packages/react-native/ReactAndroid/src/main/java/com/facebook/react/fabric/events/FabricEventEmitter.kt

```diff
@@ -16,7 +16,7 @@ import com.facebook.react.uimanager.events.RCTModernEventEmitter
 import com.facebook.systrace.Systrace
 
 internal class FabricEventEmitter(private val uiManager: FabricUIManager) : RCTModernEventEmitter {
-  @Deprecated("Deprecated in Java")
+  @Deprecated("Deprecated in Java", ReplaceWith("receiveEvent(ViewUtil.NO_SURFACE_ID, targetTag, eventName, params)"))
```

Comment:
This is not the right replacement. NO_SURFACE_ID will likely cause the event to be dropped.

Reply:
Thanks for pointing out @javache. I've fixed the replacement to suggest passing the surfaceId instead in ea87ab75bc585ce6023db1ccfb490434b9193ccf. Would that be a correct replacement?